docs/index.md

```diff
@@ -105,35 +137,34 @@ class Settings(BaseSettings):
 ```
 
 When `case_sensitive` is `True`, the environment variable names must match field names (optionally with a prefix),
-so in this example
-`redis_host` could only be modified via `export redis_host`. If you want to name environment variables
+so in this example `redis_host` could only be modified via `export redis_host`. If you want to name environment variables
 all upper-case, you should name attribute all upper-case too. You can still name environment variables anything
 you like through `Field(validation_alias=...)`.
 
-In Pydantic **v1** `case_sensitive` is `False` by default and all variable names are converted to lower-case internally.
-If you want to define upper-case variable names on nested models like `SubModel` you have to
-set `case_sensitive=True` to disable this behaviour.
+!!! warning
+    In Pydantic **V1** `case_sensitive` is `False` by default and all variable names are converted to lower-case internally.
```

Comment:
```suggestion
    The config option `case_sensitive` is `False` by default and all variable names are converted to lower-case internally.
```
I think we shouldn't mention V1 any more, unless it's for the sake of contrasting v1 and v2

Reply:
It's supposed to be exactly that: a contrast between v1 and v2. 🤔

Reply:
if that's the case, change  it to past tense — "In V1 case_sensitive _was_ false"

Reply:
but the other thing that's weird is that this is pydantic-settings, so saying in pydantic-v1 is a bit weird. I'd maybe say, "In Pydantic V1, when `BaseSettings` was provided in the Pydantic library, ..."

Reply:
```suggestion
    In Pydantic V1, when `BaseSettings` was provided by the Pydantic library, `case_sensitive` was `False`
    by default, and all variable names were converted to lower-case internally.
```